source/release-notes/1.8.txt

```diff
@@ -21,9 +21,83 @@ Release Notes for mongosync 1.8
 
 **Upcoming**
 
-New Features:
+Updates to ``mongosync`` ``/progress`` API endpoint
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Returns a top-level boolean ``success`` field.
+
+- Returns the integer ``totalEventsApplied`` field, which approximates 
```

Comment:
I think we can just say something like `Reports a ``totalEventsApplied`` field`
and add the details in an entry on the [/progress page]([mongodb.com/docs/cluster-to-cluster-sync/current/reference/api/progress](https://www.mongodb.com/docs/cluster-to-cluster-sync/current/reference/api/progress/)) below `lagTimeSeconds`

Reply:
Shortened release notes message based on your suggestion.
The progress response is updated in DOCSP-41853 ([direct link to file diff](https://github.com/mongodb/docs-cluster-to-cluster-sync/pull/362/files#diff-1f93729bd80ce7e29998a262f86151b2aedf90c1c13d0b27d1896e41e11080fb))